employee-rostering-server/src/main/resources/org/optaweb/employeerostering/server/solver/employeeRosteringScoreRules.drl

```diff
@@ -89,6 +91,64 @@ rule "No 2 shifts within 10 hours from each other"
         scoreHolder.addHardConstraintMatch(kcontext, -1);
 end
 
+rule "Daily minutes must not exceed contract maximum"
+    when
+        $employee : Employee($contract : contract, $contract.getMaximumMinutesPerDay() != null)
+        $s : Shift(employee == $employee, $startDateTime : startDateTime)
+        Number( intValue > $contract.getMaximumMinutesPerDay() ) from accumulate(
+            Shift(employee == $employee, $shiftStart : startDateTime,
+            $shiftEnd : endDateTime,
+            DateTimeUtils.dayOf($shiftStart) == DateTimeUtils.dayOf($startDateTime)),
```

Comment:
Remove the class DateTimeUitils, all those methods exist in java.time already.

For example:
- Check if 2 dateTimes are the same day:
-- shiftStart.toLocalDate().equals(startDateTime.toLocalDate())
- number of days between 2 dates:
  https://stackoverflow.com/questions/27005861/calculate-days-between-two-dates-in-java-8